index.html

```diff
@@ -4,14 +4,22 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/home.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
     <title>Hostal las paisas</title>
 </head>
 <body>
     <nav>
          <!-- Navigation section -->
     </nav>
-    <section>
+    <section class="homepage_hero">
         <!-- Hero section, the cover for the page -->
```

Comment:
now that this section has content this comment should be moved above the `section` tag or be removed

Reply:
Also for consistency with our CSS i advise 2 spaces indentation in HTML as well.
This is just a suggestion and does not represent a blocker.